utils/openshift_acme.py

```diff
@@ -1,76 +1,31 @@
 ACME_DEPLOYMENT = """
-apiVersion: extensions/v1beta1
 kind: Deployment
+apiVersion: apps/v1
 metadata:
   name: %(deployment_name)s
   labels:
     app: openshift-acme
 spec:
-  progressDeadlineSeconds: 600
-  replicas: 1
-  revisionHistoryLimit: 10
+  replicas: 2
```

Comment:
we probably don't need the extra replica here? As long as the pod is up "most of the time" it will be able to handle new certs, renewals, etc...

Reply:
done